src/style.css

```diff
@@ -1 +1,90 @@
-/* add styles here */
+body {
+  font-family: ' Avenir', Arial, sans-serif;
```

Comment:
There's an unnecessary space before 'Avenir' in the font-family declaration. It should be removed to maintain clean and consistent code formatting.